client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-identify/index.tsx

```diff
@@ -96,8 +96,12 @@ const SiteMigrationIdentify: Step = function ( { navigation, variantSlug } ) {
 	);
 
 	const urlQueryParams = useQuery();
+
 	const shouldHideBack = () => {
-		const isEntrepreneurSignup = urlQueryParams.get( 'ref' ) === 'entrepreneur-signup';
+		const isEntrepreneurSignup = [ 'entrepreneur-signup', 'wp-admin-importer' ].includes(
```

Comment:
should this flag be called something else?
```suggestion
		const shouldHideBasedOnRef = [ 'entrepreneur-signup', 'wp-admin-importer' ].includes(
```

Reply:
Logic improved here 
5b01822